mythril/laser/ethereum/call.py

```diff
@@ -122,17 +122,32 @@ def get_call_data(global_state, memory_start, memory_size, pad=True):
     :return: Tuple containing: call_data array from memory or empty array if symbolic, type found
     """
     state = global_state.mstate
+    transaction_id = '{}_internalcall'.format(global_state.current_transaction.id)
     try:
         # TODO: This only allows for either fully concrete or fully symbolic calldata.
         # Improve management of memory and callata to support a mix between both types.
-        call_data = state.memory[util.get_concrete_int(memory_start):util.get_concrete_int(memory_start + memory_size)]
-        if len(call_data) < 32 and pad:
-            call_data += [0] * (32 - len(call_data))
+        calldata_from_mem = state.memory[util.get_concrete_int(memory_start):util.get_concrete_int(memory_start + memory_size)]
```

Comment:
How about something like this?
```
starting_calldata = [state.memory[i] for i in range(memory_start, memory_start + memory_size)]:
```

Reply:
Of with concrete variables ^^

Reply:
This will not work, because memory isn't always byte indexed, so I need some hack to catch 256 bit BitVecs that are in memory.

Reply:
This will not work, because memory isn't always byte indexed, so I need some hack to catch 256 bit BitVecs that are in memory.

Reply:
ok, for documentation: with https://github.com/ConsenSys/mythril-classic/issues/563 the solution would work